src/ottplan.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Scanner;
+import java.util.HashMap;
+import java.util.Map;
+abstract class Subscription{
+    public enum namesOfPlans{
+        prime("PRIME"), premium("PREMIUM"), platinum("PLATINUM");
+        private final String stringvalue;
+        namesOfPlans(final String s)
+        {
+            stringvalue=s;
+        }
+        public String getValue()
+        {
+            return stringvalue;
+        }  
+    }
```

Comment:
```suggestion
    public enum Plan{
        PRIME,
        PREMIUM,
        PLATINUM;
    }
```

Plan.PRIME.name() would give the string value of the enum. Name is not needed.